lib/engine/g_1856/corporation.rb

```diff
@@ -56,6 +59,27 @@ def release_escrow!
         @capitalization = :incremental
       end
 
+      # Issue more shares
+      def issue_shares!
+        if total_shares == 10 # was 10 share
+          @log << "#{@name} shares are 5% shares"
+
+          @all_shares.each_with_index do |share, index|
+            share.percent = index.zero? ? 10 : 5
+            share.cert_size = index.zero? ? 1 : 0.5
+          end
+        end
+
+        @log << "#{@name} issues 10 more shares"
+        num_shares = @all_shares.count
+        # Yes, this can create 50 5% shares in one corp. 1891 does this. It is weird
```

Comment:
50 5% shares? Isn't 20 max in 1856?

Reply:
Yeah 20 is max in 1856. 1891 (a 1856 variant) lets the national keep issuing shares. it handles paying dividends by taking the run into the treasury then paying out if the treasury has enough to pay dividends, otherwise it keeps the money

Reply:
It gets a stock bump if it pays out and if it doesn't have enough to pay out it spins in place